silkworm/silkrpc/core/gas_price_oracle.cpp

```diff
@@ -32,7 +32,7 @@ struct PriceComparator {
     }
 };
 
-Task<intx::uint256> GasPriceOracle::suggested_price(uint64_t block_number) {
+Task<intx::uint256> GasPriceOracle::suggested_price(BlockNum block_number) {
```

Comment:
The same change should be applied also in `gas_price_oracle.hpp`

Reply:
ok